shell/artifacts/Social/source/PostTimeRanker.js

```diff
@@ -0,0 +1,31 @@
+// @license
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+
+defineParticle(({Particle}) => {
+  return class PostTimeRanker extends Particle {
+    setViews(views) {
+      this.on(views, 'input', 'change', e => {
```

Comment:
From local testing it appears that if one views a feed with varying data sorted correctly, and then in another tab one adds a new post that needs to be sorted amidst the existing, it wasn't showing up in the right order. I'll look at this more closely separately but wanted to mention in case you have thought on what might cause. In general the shell updates `BOXED_posts` in response to new ones coming into shared existence, and that should trigger this particle to re-sort and its output to update the input to `List` and `PostMuxer`, so I would have expected it to work.

Reply:
Ah, I think I need a `RankedList` and `RankedPostMuxer` (conceptually at least) that actually sort the posts by rank before working with them. Will do separately.